article_maker.cc

```diff
@@ -151,13 +155,12 @@ std::string ArticleMaker::makeHtmlHeader( QString const & word,
 
   if( GlobalBroadcaster::instance()->getPreference()->darkReaderMode )
   {
-    // #242525 because Darkreader will invert pure white to this value
+    // Why .gdarticle background reset?
+    // some custom theme may change it to other colors and they looks horrible in darkreader mode
+
     result += R"(
 <script src="qrc:///scripts/darkreader.js"></script>
-<style>
-body { background: #242525; }
-.gdarticle { background: initial;}
-</style>
+<style> .gdarticle { background: initial;} </style>
```

Comment:
try set the background to transparent ,and have a try.

set the transparent color can also be applied when the darkreadermode is off.

Reply:
the white flash also occured without dark reader mode,
it only more significant in dark reader mode

Reply:
> set the transparent color can also be applied when the darkreadermode is off.

A few themes set the .gdartile to explicit yellow. We just want to reset them in darkreader mode, so we cannot do that.